deploy/helm/sumologic/conf/setup/setup.sh

```diff
@@ -1,5 +1,18 @@
 #!/bin/bash
 
+readonly DEBUG_MODE=${DEBUG_MODE:="false"}
+readonly DEBUG_MODE_ENABLED_FLAG="true"
+
+if [[ ${DEBUG_MODE,,} == ${DEBUG_MODE_ENABLED_FLAG} ]]; then
```

Comment:
What are the `,,` for?

Reply:
It's an 'ignore case' - it lowers the value so we can compare ignoring True vs true vs TRUE

Reply:
Let me add a comment in the code.